{{ cookiecutter.repo_name }}/.pre-commit-config.yaml

```
@@ -0,0 +1,14 @@
+repos:
+-   repo: https://github.com/ambv/black
+    rev: stable
+    hooks:
+    - id: black
+      language_version: python3.7
+-   repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v2.0.0
+    hooks:
+    - id: flake8
+-   repo: https://github.com/kynan/nbstripout
+    rev: 0.3.9
+    hooks:
+    - id: nbstripout
```

Slightly better YAML formatting would help here:
```suggestion
  - repo: https://github.com/ambv/black
    rev: stable
    hooks:
      - id: black
        language_version: python3.7
  - repo: https://github.com/pre-commit/pre-commit-hooks
    rev: v2.0.0
    hooks:
      - id: flake8
  - repo: https://github.com/kynan/nbstripout
    rev: 0.3.9
    hooks:
      - id: nbstripout
```